frontend/src/scenes/experiments/experimentLogic.tsx

```diff
@@ -292,10 +292,10 @@ export const experimentLogic = kea<experimentLogicType>({
             }
         },
         launchExperiment: () => {
-            actions.updateExperiment({ start_date: dayjs() })
+            actions.updateExperiment({ start_date: dayjs().format('YYYY-MM-DD') })
```

Comment:
this is dangerous, since it means it's losing data (starting too early / ending too late) - we need the time info too here to be accurate :)

Reply:
(or possibly an existing bug that's now easier to see 👀 ) - I suspect this xD

Reply:
Maybe we could just set the time/date on the backend instead? Seems more fault proof that way too

Reply:
either way i updated it on the frontend here https://github.com/PostHog/posthog/pull/8208

Reply:
Good point! I think we still don't account for timezone problems that are bound to show up, except in London 😂  - will check this out